spec/poros/quote_search_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe QuoteSearch do 
+  before :each do 
```

Comment:
Nice job getting data in here to test against. You could store that in a fixture file and the do a file.read. But for a timed assessment like this, you could also pass the hash the way it is at the PORO level. I don't think that you have all of these fields in your PORO.